exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -5,12 +5,15 @@
 
 namespace TriviaGame
 {
-    public class Game
+    public class Trivia_Game
     {
-        List<string> players = new List<string>();
+        List<string> listOfAllPlayers = new List<string>();
 
-        int[] places = new int[6];
-        int[] purses = new int[6];
+        int[] playersPosition = new int[6];
+        int[] playersPurses = new int[6];
+        int[] popArray = { 0, 4, 8 };
```

Comment:
That's good, I didn't think to store the numbers in an array. Pretty straightforward.